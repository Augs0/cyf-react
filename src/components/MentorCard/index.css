.mentor {
  margin-bottom: 2em;
}

.mentor-image {
  max-width: 100%;
}

.mentor-name {
  margin-bottom: 0.2em;
}

.mentor-first-name::before, .mentor-first-name::after {
  display: inline;
  color: #d12f2f;
  font-weight: bold;
}

.mentor-first-name::before {
  content: '<';
}

.mentor-first-name::after {
  content: '> ';
}

.mentor-title {
  color: gray;
}
