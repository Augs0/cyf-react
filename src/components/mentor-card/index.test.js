import React from 'react';
import { shallow } from 'enzyme';

import MentorCard from './';

describe('MentorCard Component', () => {
  it('renders correctly', () => {
    const mentor = {
      firstName: 'Germ√°n',
      lastName: 'Bencci',
      photo: 'german.jpg',
      title: 'Organiser'
    };
    expect(shallow(<MentorCard {...mentor} />)).toMatchSnapshot();
  });
});
