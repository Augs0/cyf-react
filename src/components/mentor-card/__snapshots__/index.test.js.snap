// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MentorCard Component renders correctly 1`] = `
<Styled(div)>
  <Styled(img)
    alt="Germán Bencci"
    src="german.jpg"
  />
  <Styled(h4)>
    <Styled(span)>
      Germán
    </Styled(span)>
    <span>
      Bencci
    </span>
  </Styled(h4)>
  <Styled(div)>
    Organiser
  </Styled(div)>
</Styled(div)>
`;
